great_expectations/analytics/client.py

```diff
@@ -56,7 +57,13 @@ def init(  # noqa: PLR0913 # FIXME CoP
         conf["organization_id"] = organization_id
     if oss_id:
         conf["oss_id"] = oss_id
-    update_config(config=Config(cloud_mode=cloud_mode, **conf))
+    update_config(
+        config=Config(
+            cloud_mode=cloud_mode,
+            user_agent_str=user_agent_str,
```

Comment:
mypy was upset when I put the string in `conf`. I think we can probably clean this method up at some point - there's a bunch of logic that doesn't really impact behavior.